src/website/app/demos/Icon/examples/title.js

```diff
@@ -20,8 +20,9 @@ import IconSentimentSatisfied from '../../../../../Icon/IconSentimentSatisfied';
 export default {
   id: 'title',
   title: 'Title',
-  description:
-    'Standalone icons need a title, as opposed to decorative icons (those that repeat the information conveyed by text or do not add significant value).',
+  description: `Standalone icons need a title, as opposed to decorative icons (those that repeat the information conveyed by text or do not add significant value).
+Mineral UI generates the hooks for assistive technology under the covers after you set the \`title\` attribute here.
```

Comment:
"under the covers" seems off-tone for us.

"...generates the code for assistive technology after you..." instead?

Reply:
this is a fair assesment